### PR TITLE
#2 fix import templates

### PR DESCRIPTION
**Pull request by tuanlc**
Fix template import

This PR cache not only templates that are used by `templateUrl` but also the dynamic/ng-include templates like:

```
inbox-subheader-more-button(action-list="/unifiedinbox/views/thread/view/action-list.html")
```

**TODO:**

- [x] Fix the include syntax in pug file
```
include jmap-empty-message.pug

.message.inbox-dnd-info-message.all-centered.hidden-xs
  i.mdi.mdi-lightbulb-outline
  span {{ 'Did you know you can drag & drop messages to this folder? Give it a try!' | translate }}
```

### CODE REVIEW

File: src/linagora.esn.unifiedinbox/app/directives/main.js [hunk 1]
```diff
@@ -387,19 +387,6 @@ angular.module('linagora.esn.unifiedinbox')
     };
   })
 
-  .directive('emailBodyEditor', function(emailBodyService) {
```
Comment (MichaelBailly) [inline src/linagora.esn.unifiedinbox/app/directives/main.js:390]: Why is that removed ?

File: src/linagora.esn.unifiedinbox/app/directives/subheaders.js [hunk 2]
```diff
@@ -39,13 +39,6 @@ angular.module('linagora.esn.unifiedinbox')
     };
   })
 
-  .directive('fullscreenEditFormSubheader', function() {
```
Comment (MichaelBailly) [inline src/linagora.esn.unifiedinbox/app/directives/subheaders.js:42]: Why is that removed ?

File: src/linagora.esn.unifiedinbox/app/directives/main.js (hunk 1 above)
Comment (tuanlc) [reply src/linagora.esn.unifiedinbox/app/directives/main.js:390]: Because it is not used anymore. And the template has been not exist. This belong to the commit that I explained.

File: src/linagora.esn.unifiedinbox/app/directives/subheaders.js (hunk 2 above)
Comment (tuanlc) [reply src/linagora.esn.unifiedinbox/app/directives/subheaders.js:42]: Because it is not used anymore. And the template has been not existed. This belongs to the commit that I explained.
